routes/Login/createUser.js

```diff
@@ -66,6 +56,28 @@ router.route("/").post(function(req, res) {
 
             res.status(503).send("ERROR: " + err);
         })
+        .then(function() {
+            require('../../imp_services/implogging')(req.cookies.IMPId, function(logService){
+                logService.action.value = username;
+                logService.setType(800);
+                logService.store(function(err, results){
+                    if(err){
+                        res.status(500).send(err);
+                    }
+                    else{
+                        SendConfirmation(email, function (err) {
```

Comment:
Thats the point here, the SendConfirmation is handled in its own function, but shouldnt be sent until after the store happens.